config/application.yml.example

```diff
@@ -33,6 +33,8 @@ stale_session_window: '180'
 support_email: 'hello@login.gov'
 support_url: 'https://18f.gov/contact'
 
+valid_authncontexts: '["http://idmanagement.gov/ns/assurance/loa/1", "http://idmanagement.gov/ns/assurance/loa/3"]'
```

Comment:
since "authentication" and "contexts" are separate words, I think this would be more readable as `valid_authn_contexts`